man/tpm2_nvread.1.md

```diff
@@ -34,7 +34,7 @@
   * **-f**, **--out-file**=_FILE_:
     file to write data
 
-  * **-P**, **--auth-hierarchy**=_HIERARCHY\_AUTH_:
+  * **-P**, **--hierarchy**=_HIERARCHY\_AUTH_:
```

Comment:
-a is --hierarchy, for -P is --auth-hierarcy-value. Although probably should be changed to --auth-hierarcy for consistency with the other tools?